### PR TITLE
add book page

### PR DESCRIPTION
**Pull request by ZPoreba**

### CODE REVIEW

File: client/src/BookProfile/BookProfile.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+const { Meta } = Card;
+
+export class BookProfile extends Component {
+
+    constructor(props) {
+        super(props);
+
+        let splitedPath = window.location.pathname.split("/");
+        let bookId = 1;
```
Comment (SzymonSmykala) [inline client/src/BookProfile/BookProfile.js:17]: why do we use hardcoded id, instead of using props?

Comment (ZPoreba) [reply client/src/BookProfile/BookProfile.js:17]: Its not hardcode it's default. If I won't set default then when somebody type incorrect  url e.g without id of book at the end there will be problem

Comment (SzymonSmykala) [reply client/src/BookProfile/BookProfile.js:17]: please rename it to defaultBookId then, if possible move it to const field.

Comment (SzymonSmykala) [reply client/src/BookProfile/BookProfile.js:17]: ok, leave bookId as it is state. Just create const with "1" inside.

Comment (ZPoreba) [reply client/src/BookProfile/BookProfile.js:17]: and then in Line 18 I will assign value which is not default to variable default ?? And to const ? hmmm

Comment (SzymonSmykala) [reply client/src/BookProfile/BookProfile.js:17]: no
`let bookId = defaultBookId;`

Generally if you have any magic number or string it should have a name

File: client/src/BookProfile/BookProfile.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    componentDidMount() {
+        this.loadBookData();
+    }
+
+    loadBookData() {
+        let bookData = Object.keys(data).filter(item => { return data[item].id === this.state.bookId })[0];
+        bookData = data[bookData];
+
+        if(bookData.image.length === 0) bookData.image = defaultBook;
```
Comment (SzymonSmykala) [inline client/src/BookProfile/BookProfile.js:35]: I would recommend using curly braces everywhere. Anyway, is that some kind of standard to not using it?

Comment (ZPoreba) [reply client/src/BookProfile/BookProfile.js:35]: Well if it's needed with one line after condition I can add it. Is there standard to use it ?

Comment (SzymonSmykala) [reply client/src/BookProfile/BookProfile.js:35]: Uncle Bob - Clean Code

Comment (ZPoreba) [reply client/src/BookProfile/BookProfile.js]: Good to know

Comment (SzymonSmykala) [reply client/src/BookProfile/BookProfile.js]: https://www.c-sharpcorner.com/UploadFile/d0e913/lame-question-of-the-day-role-of-curly-braces-in-our-cod/

**Pull request merged**